### ISSUE TITLE
Can I use imported objects without package name?

### ISSUE THREAD

**Issue by puppuccino** [veryl-lang/veryl#481]
I have a package defined in a file:
```
package my_pkg {
    localparam foo: u32 = 42;
}
```
I want to use it in other files like:
```
import my_pkg::*;
module (
...
) {
localparam bar: u32 = foo;
// localparam bar: u32 = my_pkg::foo; <- it will compile if you write it like this
}
```
but this code produces an undefined error at `foo` and fails to compile. Is this a bug? If not, any plans to implement this feature?

**Comment by dalance** [comment]
Thank you for your report.
I found it, but don't have resolved yet.
I'll prioritize it.

Related: #126

**Comment by dalance** [comment]
This issue was resolved by #511.
